gno.land/pkg/gnoweb/views/faucet.html

```diff
@@ -20,7 +20,7 @@
           {{ end }}
         </script>
 
-        <form method="POST" action="{{ .Data.faucetURL }}">
+        <form method="POST" action="{{ .Data.Faucet.faucetURL }}">
```

Comment:
Side-note: what a horrible templating engine we use.

Have you tested this works? Reding through the code it shouldn't, this change should:

```suggestion
        <form method="POST" action="{{ .Data.Config.FaucetURL }}">
```

Reply:
OH Damn!

You're half right, i'm using gnolang/faucet, which use different variable name 😮‍💨